### DISCUSSION TITLE
Meeting  alignment and font questions

### DISCUSSION THREAD

**Discussion by ACAwebbuilder** [code4recovery/12-step-meeting-list#1227] [Q&A] [answered]
Good morning everyone ~

I just installed this plug-in and the table of meetings on the main meeting page (as well as information on individual meeting pages) is aligned left, as if I have a right sidebar, which I don't think I do (all the pages I have created don't have a right sidebar, I have removed all the default widgets I could find, and and checked the customizer settings for Blog and Post and removed any sidebar defaults I could find.  Here is a [link](http://rvaacadf.org/meetings/) to the main meeting page.  We don't have SSL installed yet, so you might get a warning about that.

I  built the site using the Astra theme and it has no page editors installed. I have used the plug-in before with other themes and sites using Elementor and Beaver builder and haven't had this issue.  Does anyone have any thoughts?

Also, the title for the individual meetings is using the H2 font, which is a unique one on our site and not what we would want the title to use.  Any ideas on how to change which Heading is used?  Or do I need to just do this in CSS?

Thanks!

**Comment by joshreisner** [comment, marked as answer]
hi @ACAwebbuilder - looks like your theme is applying the left-aligned layout, TSML UI will just fill the container it's put inside. 

at any rate, CSS is the way to go! to center it you should be able to do:

```css
body.tsml-ui .wp-site-blocks {
  margin: 0 auto;
}
```

TSML UI uses h2 for semantic reasons, and that can't be overridden, but i can probably reset that font on my end to use the default font. (in the meantime you could also override that with CSS)

**Reply by joshreisner** [reply]
@ACAwebbuilder great! to the width question, add a few more rules:

```css
body.tsml-ui .wp-site-blocks {
  margin: 0 auto;
  max-width: 1170px;
  width: 100%;
}
```

tweak that max-width as you see fit, shouldn't break mobile 🤞

**Reply by ACAwebbuilder** [reply]
That does it! Thanks so much Josh!

**Comment by ACAwebbuilder** [comment]
Thank you Josh!  This is very helpful and did what I needed.

It widened the main meeting page, as well, which is great.  Do you have any thoughts as to how to widen the individual meeting pages?  I added a "width" to the body.tsml-ui .wp-site-blocks CSS you gave me above and that did it, but it made the main meeting page more narrow.  I would like the width to be wide enough that all text and boxes are as wide as they need to be for the text.  Surprisingly, mobile view is perfect, so I don't want to affect that.

Thanks again!
